db_test.go

```diff
@@ -1373,6 +1373,85 @@ func TestDBUnmap(t *testing.T) {
 	db.DB = nil
 }
 
+// Ensure that a database cannot exceed its maximum size
+// https://github.com/boltdb/bolt/issues/928
```

Comment:
It seems https://github.com/boltdb/bolt/issues/928 doesn't exist?

Reply:
I have edited all comments related to this issue to point to the correct issue